main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -52,22 +58,22 @@ const aliceMoola = await E(aliceMoolaPurse).withdraw(threeMoola);
 const alicePayment = { Asset: aliceMoola };
 ```
 
-In order for Alice to escrow with Zoe she needs to use her invite.  Once
-Alice uses her invite and makes her offer she will receive a `seat`, which
-gives her access to the outcome of the offer and to her payouts.
+For Alice to escrow with Zoe, she needs to use her invitation.  
+Then she makes her offer and receives a `seat`. The `seat`
+gives her access to the offer's outcome and her payouts.
```

Comment:
```suggestion
gives her access to the offer's result and her payouts.
```
(previously `outcome` was the name of a property. Now `getOfferResult` is the method to get that value, so if we have the choice to use `result` rather than `outcome`, I think we should use `result`, all things being equal.